src/types/atproto/index.ts

```diff
@@ -0,0 +1,26 @@
+export * as profile from '#/types/atproto/profile'
```

Comment:
NIT: While `dangerousIsType` is indeed "Atprotoy" (has to do with the way atproto allow "open" data), the `profile` &  `starterPack` are not really part of "atproto".

Wouldn't it make more sense to name this something like `bsky` or `api` (contextualized by the app) ?

Reply:
Oh yeah that's a solid idea 👍